### PR TITLE
docs: improve cms-mc-production-overview

### PR DESCRIPTION
**Pull request by heitorPB**
Changes due to @katilp feedback

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/cms-mc-production-overview/cms-mc-production-overview.md [hunk 1]
```diff
@@ -1,227 +1,234 @@
-The following diagram shows an overview of analysis in CMS. The arrows describes the direction of the flow of information.
+The CMS data files have gone through several processing steps before they are in format good for analysis. The following diagram shows an overview of these step. The arrows describe the direction of the flow of information.
```
Comment (katilp) [inline cernopendata/modules/fixtures/data/docs/cms-mc-production-overview/cms-mc-production-overview.md]: these step -> these steps

**Comment by heitorPB** [comment]
Fixed!

**Comment by heitorPB** [comment]
related issues: #1398 #2249

**Comment by heitorPB** [comment]
@katilp could you review the last additions, please?

**Pull request merged**